### ISSUE TITLE
*.a11y.js test are not running.

### ISSUE THREAD

**Issue by nahumzs** [acl-services/paprika#62]
# Help wanted
## Please write your question.
Currently, there couple component with a11y.js test in place but are not running because the extension should be spec.js and as well axe is not connected correctly.

Also worth to investigate if using jest-a11y is the best way to test accessibility because is not checking each time the test is interacting with the UI, maybe would be to use something like cypress axe where we can test the interaction and test that accessibility is maintained correctly. cc @oscarkwan 

## Additional context

https://github.com/acl-services/paprika/pull/73#issuecomment-490275068

**Comment by oscarkwan** [comment]
Closing this, done here https://github.com/acl-services/paprika/pull/979
